spec/fixtures/content_store/historic_appointments.json

```diff
@@ -0,0 +1,204 @@
+{
```

Comment:
We should perhaps use this data in the tests if we're going to add a fixture - e.g. asserting that the right people appear in the right places